TerminalDocs/customize-settings/interaction.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Windows Terminal Interaction Settings
+description: Learn how to customize interaction settings within Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 1/28/2021
+ms.topic: how-to
+ms.localizationpriority: high
+---
+
+# Interaction settings in Windows Terminal
+
+The properties listed below affect the entire terminal window, regardless of the profile settings. These should be placed at the root of your settings.json file.
+
+> [!TIP]
+> If you'd like to edit these settings using the settings UI, you will have to add the `"openSettings"` action to your `"actions"` array in order to open it with the command palette or keyboard.
+
+```json
+{ "command": { "action": "openSettings", "target": "settingsUI" }, "keys": "ctrl+shift+s" },
+```
+
+> [!IMPORTANT]
+> The settings UI is only available in [Windows Terminal Preview](https://aka.ms/terminal-preview).
+
+## Automatically copy selection to clipboard
+
+When this is set to `true`, a selection is immediately copied to your clipboard upon creation. The right-click on your mouse will always paste in this case. When it's set to `false`, the selection persists and awaits further action. Using your mouse to right-click will copy the selection.
```

Comment:
```suggestion
When this is enabled, a selection is immediately copied to your clipboard upon creation. The right-click on your mouse will always paste in this case. When it's disabled, the selection persists and awaits further action. Using your mouse to right-click will copy the selection.
```
Maybe? To start steering away from json?
(same sentiment applies elsewhere)

Reply:
I don't want to steer away from JSON until we set the settings UI as the default behavior.